### PR TITLE
Features/two score

### PR DESCRIPTION
**Pull request by zbrotherton**
One cube load needs to be merged before this PR is reviewed

### CODE REVIEW

File: src/main/java/frc/robot/classes/Auton.java [hunk 1]
```diff
@@ -344,15 +353,28 @@ private Command getAutonSequence(){
...
+                    new SequentialCommandGroup(
+                        resetOdometry("2ScoreCube-1"),
+                        cubeapultSubsystem.launch(),
+                        new FollowPathWithEvents(
+                            swerveSubsystem.followTrajectory("2ScoreCube-1", getTrajectory("2ScoreCube-1")), 
+                            getTrajectory("2ScoreCube-1").getMarkers(),
+                            actions
                         ),
-                        highScoreSequenceCube()
+                        new StartEndCommand(() -> swerveSubsystem.setModuleStates(swerveSubsystem.convertToModuleStates(0, -0.1, 0)), () -> swerveSubsystem.stopModules()).withTimeout(0.3),
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java]:
```suggestion
                        translateY(-0.1, 0.3),
```

File: src/main/java/frc/robot/classes/Auton.java [hunk 2]
```diff
@@ -344,15 +353,28 @@ private Command getAutonSequence(){
...
-                        .alongWith(intakeSubsystem.pullTimed(1, false)
-                            .andThen(armSubsystem.goToStateDelay(ArmState.STOWED))
+                        resetOdometry("2Score-1"),
+                        cubeapultSubsystem.launch(),
+                        new FollowPathWithEvents(
+                            swerveSubsystem.followTrajectory("2Score-1", getTrajectory("2Score-1")), 
+                            getTrajectory("2Score-1").getMarkers(),
+                            actions
+                        ),
+                        new StartEndCommand(() -> swerveSubsystem.setModuleStates(swerveSubsystem.convertToModuleStates(0, -0.1, 0)), () -> swerveSubsystem.stopModules()).withTimeout(0.3),
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java]:
```suggestion
                        translateY(-0.1, 0.3),
```

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 3]
```diff
@@ -536,6 +540,22 @@ public void stopsim() {
...
+  public Rotation3d getElbowRotation3d() {
+    return new Rotation3d(getElbowAngle(), 0, 0);
+  }
+
+  public Pose3d getBasePose3D() {
+    return new Pose3d(0.008083, -0.173961, 0.20541, getBaseRotation3d());
+  }
+
+  public Pose3d getElbowPose3d() {
+    return new Pose3d(getArmX(), getArmY(), -0.042619, getElbowRotation3d());
```
Comment (azvanderpas) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java]: Magic numbers should be constants or at least have comments to explain where they come from

**Review by azvanderpas — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java (hunk 3 above)
Comment (Shazamboo) [reply src/main/java/frc/robot/subsystems/ArmSubsystem.java]: numbers are offsets from the CAD model in blender and I added comments explaining them.

**Review by azvanderpas — APPROVED** [review]

**Pull request merged**